otherlibs/stdune-unstable/set.ml

```diff
@@ -54,6 +54,16 @@ module Make (Key : Map_intf.Key) (M : Map_intf.S with type key = Key.t) = struct
 
   let union_all l = union_map l ~f:(fun x -> x)
 
+  let combine = union
```

Comment:
We have the `Monoid.Union` functor, so you can probably just `include` the corresponding monoid interface here.

Reply:
(Alternatively, we could also add `Monoid.Set`, similar to `Monoid.Appendable_list`.)

Reply:
(Added some time later) Actually, it may be best not to commit to only one monoid instance for `Set` here. In addition to using `combine = union`, one might also want to use `combine = intersection` in some situations.

Reply:
I see. Alright, let's use `Monoid.Union` then

Reply:
done